core/src/main/java/tc/oc/pgm/action/WeatherType.java

```diff
@@ -0,0 +1,7 @@
+package tc.oc.pgm.action;
```

Comment:
housing this enum standalone is probably overkill, consider just putting it inside weather action, or even weather match module